recipe/yum_requirements.txt

```diff
@@ -0,0 +1,9 @@
+mesa-libGL
+mesa-dri-drivers
+libselinux
+libXdamage
+libXfixes
+libXxf86vm
+libxcb
+expat
```

Comment:
Our `expat` is not OK?
PS: many of these can become `ctd`s.

Reply:
Sorry, I just copied from `gnuplot` without thinking...

> Note that yum_requirements.txt will go away at some point

Cool. Let me know

Reply:
> Sorry, I just copied from `gnuplot` without thinking...

No problem. That is what I would do too.

> > Note that yum_requirements.txt will go away at some point
> 
> Cool. Let me know

We just need to get what are the corresponding `ctd`s for the X stuff here and try our expat.